one/src/events.rs

```diff
@@ -156,3 +376,215 @@ async fn migrate_from_database(input_ceramic_db: PathBuf, store: SQLiteBlockStor
     );
     result.unwrap()
 }
+
+#[derive(Debug, Clone)]
+pub struct SQLiteRootStore {
+    pool: SqlitePool,
+}
+
+impl SQLiteRootStore {
```

Comment:
Can we add some comments here on what this is storing and how it is used please?